src/formatters/html.js

```diff
@@ -20,6 +20,10 @@ function htmlEscape(text) {
   return html;
 }
 
+HtmlFormatter.prototype.prepareContext = function(context) {
+  BaseFormatter.prototype.prepareContext.call(this, context);
+};
```

Comment:
mm sorry I don't think that's true, that would make js child classes extremely noisy, maybe you can provide a test case reproducing that? I guess this should be easy to test.